ui/src/components/ignite/streaming/IgniteStreamContainter.js

```diff
@@ -0,0 +1,140 @@
+import { useCallback, useEffect, useState } from 'react';
+import { customAxios as axios } from '../../helpers/customAxios';
+import {
+  VStack,
+  Box,
+  Button,
+  Stack,
+  AspectRatio,
+  Link,
+  Text,
+  Container,
+  Image,
+} from '@chakra-ui/react';
+import { ChevronLeftIcon } from '@chakra-ui/icons';
+import OnlinePageButtons from '../../sermons/OnlinePageButtons';
+import OnlinePageTabs from '../../sermons/OnlinePageTabs';
+import igniteTheme from '../igniteTheme';
+
+const IgniteStreamContainer = (props) => {
+  const { history } = props;
+  const [onlineSermon, setOnlineSermon] = useState();
+
+  const getOnlineSermon = useCallback(async () => {
+    try {
+      const { data, status } = await axios.get('/api/sermons/get-sermons');
```

Comment:
are we not gonna password protect ignite sermon? this current way of calling the sermon api implies that there is no password protection (from the  Wordpress API itself), which means that someone can go to hongkong.hmcc.net/online and watch the same stream

Reply:
yea so part of the design is that the stream page would be in /go-online, instead of /online. because people need to login first if they use /online and the crew suggests that it might be easier to give people a unique link through email that they can just click.
but now that i think abt it, i think ya this way of calling the API, we need to put the isProtected flag? and somehow need to block it from /online. maybe detect [!gnite] keyword or something